docs/rst/listeners/listeners_index.rst

```diff
@@ -6,3 +6,4 @@ Listeners
 .. toctree::
 
     /rst/listeners/domain_listener
+    /rst/listeners/physical_listener
```

Comment:
The listeners introductory page it is kind of empty, only showing the table of contents. I would write a brief introduction to this section.